Modelica/Electrical/Polyphase/Sensors/AronSensor.mo

```diff
@@ -69,17 +69,19 @@ equation
           extent={{-30,-10},{30,-70}},
           textColor={64,64,64},
           textString="W")}),      Documentation(info="<html>
-<p>
-Contains two 
-<a href=\"modelica://Modelica.Electrical.Analog.Sensors.PowerSensor\">power meters</a>
-to measure total active power in a three-phase system.
-</p>
-<p>
-This device works only for sinusoidal voltages and currents in three-phase systems without neutral.
-</p>
-<p>
-Elmar Schr&uuml;fer, Leonhard Reindl, Bernhard Zagar: Elektrische Messtechnik. Carl Hanser Verlag.<br>
-Reinhard Lerch: Elektrische Messtechnik. Springer Vieweg.
-</p>
+<p>Contains two <a href=\"modelica://Modelica.Electrical.Analog.Sensors.PowerSensor\">power meters</a> to measure total active power in a three-phase system. </p>
+<p>This device works only for in three-phase systems without neutral. </p>
+<p><br>The rationale behind this sensor is that power exchanged between two parts of a circuit is the sum of the products of currents in the <i>m</i> wires connecting the two parts times wires&apos; potentials evaluated with reference to an arbitrary potential <i>v</i><sub>ref</sub>: </p>
+<p><i>P</i>=(<i>v</i><sub>1</sub>-<i>v</i><sub>ref</sub>)*<i>i</i><sub>1</sub>+(<i>v</i><sub>2</sub>-<i>v</i><sub>ref</sub>)*<i>i</i><sub>2</sub>+... (<i>v</i><sub>m</sub>-<i>v</i><sub>ref</sub>)*<i>i</i><sub>m</sub></p>
+<p>In case of a three-phase system without neutral we may want to measure power flowing in a line, connecting say the left part of the circuit to its right part. The wires connecting the two parts are only three (i.e. <i>m</i>=3)
+ and therefore all currents between these two parts are <i>i</i><sub>1</sub>, <i>i</i><sub>2</sub>, <i>i</i><sub>3</sub>.&nbsp;</p>
+<p>Since the voltage to be taken as reference is arbitrary, we can take the voltage of conductor 2. Therefore our power becomes:</p>
+<p><i>P</i>=(<i>v</i><sub>1</sub>-<i>v</i><sub>2</sub>)*<i>i</i><sub>1</sub>+(<i>v</i><sub>2</sub>-<i>v</i><sub>2</sub>)*<i>i</i><sub>2</sub>+(<i>v</i><sub>3</sub>-<i>v</i><sub>2)</sub>*<i>i</i><sub>3 </sub>= 
+(<i>v</i><sub>1</sub>-<i>v</i><sub>2</sub>)*i<sub>1</sub>+(<i>v</i><sub>3</sub>-<i>v</i><sub>2</sub>)* i<sub>3</sub></p>
+<p>In this way, we can just sum up the power from two wattmeters to get the three-phase power.</p>
+<p>Note that this formula does not work if there are additional current paths between the left and right parts of our circuits, e.g. if both have grounds (and current flows through it).</p>
+<p>-------------</p>
+<p>For more info on why power flowing in a circuit between two subcircuits is the sum pf products of voltages times currents, the voltages being measured to n arbitrary reference potential, see </p>
```

Comment:
```suggestion
<p>For more info on why power flowing in a circuit between two subcircuits is the sum of products of voltages times currents, the voltages being measured to an arbitrary reference potential, see </p>
```

Reply:
Thank you for pointing out this misprint